common/persistence/wrappers/metered/execution_generated.go

```diff
@@ -69,71 +69,25 @@ func (c *meteredExecutionManager) Close() {
 	return
 }
 
-func (c *meteredExecutionManager) CompleteReplicationTask(ctx context.Context, request *persistence.CompleteReplicationTaskRequest) (err error) {
+func (c *meteredExecutionManager) CompleteHistoryTask(ctx context.Context, request *persistence.CompleteHistoryTaskRequest) (err error) {
 	op := func() error {
-		err = c.wrapped.CompleteReplicationTask(ctx, request)
+		err = c.wrapped.CompleteHistoryTask(ctx, request)
 		return err
 	}
 
 	if domainName, hasDomainName := getDomainNameFromRequest(request); hasDomainName {
 		logTags := append([]tag.Tag{tag.WorkflowDomainName(domainName)}, getCustomLogTags(request)...)
-		c.logger.SampleInfo("Persistence CompleteReplicationTask called", c.sampleLoggingRate(), logTags...)
+		c.logger.SampleInfo("Persistence CompleteHistoryTask called", c.sampleLoggingRate(), logTags...)
 		if c.enableShardIDMetrics() {
-			err = c.callWithDomainAndShardScope(metrics.PersistenceCompleteReplicationTaskScope, op, metrics.DomainTag(domainName),
+			err = c.callWithDomainAndShardScope(metrics.PersistenceCompleteHistoryTaskScope, op, metrics.DomainTag(domainName),
```

Comment:
since we converged into single metric name, let's include task category as a metric tag so that we still have visibility into breakdown per category

Reply:
Make sense. Let me do it in another PR.